docs/key_modules/models.md

```diff
@@ -23,6 +23,7 @@ The following table lists currently supported model platforms by CAMEL.
 | Azure OpenAI | gpt-3.5-turbo | Y |
 | OpenAI Compatible | Depends on the provider | ----- |
 | Mistral AI | mistral-large-2 | N |
+| Mistral AI | pixtral-12b-2409 | Y |
```

Comment:
Can we also add ministral 3B and 8B?

https://x.com/dchaplot/status/1846575384718172448